libhb/dvd.c

```diff
@@ -242,6 +242,22 @@ static void add_subtitle( hb_list_t * list_subtitle, int position,
             break;
     }
 
+    switch (style)
+    {
+        case HB_VOBSUB_STYLE_4_3:
+            strcat( subtitle->lang, " (4:3)" );
```

Comment:
The description shown in the CLI and GUI does have a lot of parens.  Adding different punctuation is going to look a bit strange.
e.g. 
CLI: English (Closed Caption) (Wide Screen) (iso639-2: eng) (Bitmap)(VOBSUB)
GUI: English (Closed Caption) (Wide Screen) (VOBSUB)

I was just being a good lemming.

We could probably eliminate a lot of what the CLI is showing.  I'm pretty sure it was there so the WinGui could parse out everything it needed to know when it was a CLI wrapper.

How about reformatting the whole thing to something like this?
English, Closed Caption [Wide Screen, VOBSUB]

The problem with doing this is it makes the patch much more extensive.  Everything after the "(Wide Screen)" part is added by the frontend.  So we would have to go around and change all the frontends as well.  I would suggest just having libhb build the full description and remove this extra code from the frontends.

But now this is heading in directions this patch really wasn't intended to go. So my suggestion is that reformatting should be a separate patch as well.